packages/cashier/src/pages/account-transfer/account-transfer-receipt/account-transfer-receipt.tsx

```diff
@@ -146,8 +145,6 @@ const AccountTransferReceipt = observer(({ onClose, history }: TAccountTransferR
             <Modal
                 is_open={is_switch_visible}
                 toggleModal={toggleSwitchAlert}
-                enableApp={enableApp}
-                disableApp={disableApp}
```

Comment:
@hamid-deriv I think you can also remove them at line 25 if they are not in use anymore 🤔

Reply:
Removed.